flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/NettyMessage.java

```diff
@@ -281,14 +281,18 @@ protected Object decode(ChannelHandlerContext ctx, ByteBuf in) throws Exception
 
 		final boolean isBuffer;
 
+		final boolean isCompressed;
```

Comment:
we can avoid touching this class as i mentioned via https://github.com/apache/flink/pull/10375/commits/2f538ab20262fbe9981d4851b93e9bee04fc6ad0#r353537910